wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -177,16 +177,12 @@ October 25, 2016 (11:45 UTC)
 
 Game modes must be written as follows:
 
-- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!`
 - `osu!taiko`
 - `osu!catch`
 - `osu!mania`
 
-*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
-
-Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name.
-
-Articles such as the [Ranking Criteria](/wiki/Ranking_Criteria) may use `osu!` instead of `osu!standard`.
+Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name. Do note however that citing the osu! game mode as `osu!standard` is *not* recommended — unless necesary, `osu!` should be used instead.
```

Comment:
please define "unless necessary" if you use this sentence. personally I think we may be able to rewrite the wiki in a way that it would always be clear which of the two entities (the game, or the main mode) is mentioned, so no such sentence is needed

Reply:
ah, okok, removed

Reply:
> please define "unless necessary" if you use this sentence. personally I think we may be able to rewrite the wiki in a way that it would always be clear which of the two entities (the game, or the main mode) is mentioned, so no such sentence is needed

On this point I was thinking of making a disambiguation for the term `osu!`, or if existing include this new point.

Because I'm thinking of renaming `osu!standard` > `osu!`, so I'm also thinking of a solution to not confuse the two.

Reply:
nah, what I meant is that you didn't even need to mention "osu!standard" as an alternative spelling if it will be removed (and I guess it will)

Reply:
You can call it "osu! (ruleset)" or "osu! mode" or anything. But don't call it osu!standard. It's not how I want it referenced and it sounds bad.

Reply:
@TicClick ah i see, i guess i was thinking too much when i wrote that sentence down then ;~;

removed the mention of the alternative spelling, should be good now!